app/assets/javascripts/templates/form.html

```diff
@@ -21,7 +21,7 @@
       Instructions
     </label>
     <div class="col-md-5">
-      <textarea name="instructions" class="form-control" ng-model='recipe.instructions' placeholder="e.g. Flambé for 20 seconds">
+      <textarea id="instructions" name="instructions" class="form-control" ng-model='recipe.instructions' placeholder="e.g. Flambé for 20 seconds">
```

Comment:
Do we need `id` at all?  I don't think anything is using it (we do need the fixed form tag though :)